source/database/build-nature/src/build/scripts/default.rb

```diff
@@ -0,0 +1,20 @@
+clear_log
+
+check_datasources
+
+create_database :overwrite_existing
+
+begin
+
+  import_database_structure
+  update_comments
+
+  load_data
+
+  generate_html_documentation
+  generate_rtf_documentation
+  generate_datasources_json
+
+ensure
+
```

Comment:
didn't notice this before, but not much point in having an `ensure` block if its empty?